src/RoomsSDKAdapter.integration.test.js

```diff
@@ -0,0 +1,65 @@
+import {createIntegrationSdkInstance, createIntegrationTestUser, removeIntegrationTestUser} from './testHelper';
```

Comment:
it will be cool to have this setup and teardown logic outside so all of our integration test modules have access to internal stuff by only one device registration/ socket connection, etc. Unless, it's an implicit teardown logic. just like `stopListening` from the rooms module. that should be done for each test module at least for now.  But the explicti ones should live outside in  the higher scope (device registration, mercury connection).

Reply:
I think this goes back to our conversation of having `connect/disconnect` at the `WebexSDKAdapter` level